File: Common/Entity/src/main/java/it/ey/dto/AdempimentoDTO.java
package it.ey.dto;

import it.ey.entity.Sezione22;
import it.ey.enums.TipologiaAdempimento;
import lombok.*;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class AdempimentoDTO extends  CampiTecniciDTO
{
    private Long id;

    private Long idSezione22;

    private TipologiaAdempimento tipologia;

    private String denominazione;

    private AzioneDTO azione;

    private UlterioriInfoDTO ulterioriInfo;
}

File: PIAO_FE/src/app/shared/models/classes/adempimento-dto.ts
import { TipologiaAdempimento } from '../enums/tipologia-adempimento.enum';
import { UlterioriInfoDTO } from './ulteriori-info-dto';

export class AdempimentoDTO {
  id?: number;
  idSezione22?: number;
  tipologia?: TipologiaAdempimento;
  denominazione?: string;
  azione?: UlterioriInfoDTO;
  ulterioriInfo?: UlterioriInfoDTO;

}
